src/mixins/payment-mixin.ts

```diff
@@ -28,6 +28,16 @@ export class PaymentMixin extends Mixins(ActionMixin) {
   @Getter getCurrentJsDate!: Date
   @Getter getStaffPayment!: StaffPaymentIF
   @Getter getFolioNumber!: string
+  @Getter getNr!: Partial<NameRequestI>
+
+  /** Params used to format refund message (tooltip and modal). */
+  refundParams: RefundParamsIF = {
```

Comment:
This data was OK when it was in the component, but now I think it would be better in the store. This is the first time we're storing data in a mixin and it feels wrong. Thoughts?

Reply:
I'm not 'storing' anything. I don't see any need to store it in vuex... 🤷‍♂️ Do you?

Reply:
You are storing an object with properties that may contain values, and the Existing Request component template binds to this object's properties to conditionally display text, right?

Reply:
I can store it in vuex if you prefer. I only see a real need to store in vuex when we need to share the state with other components that is not the case but ok, I will add it to vuex.

Reply:
It is shared: this mixin sets it, and the Existing Request Display component gets it.

Thanks.

Reply:
Yeah, ok. There's a case it can be shared instead of recalculating it. It makes sense. Thanks. Done.